.mypy.ini

```diff
@@ -0,0 +1,25 @@
+[mypy]
+files = a1314_message_filter.py, adapter.py, bluetooth_devices.py, hid_message_filter.py, mouse_message_filter.py
```

Comment:
agent.py, compatibility_devices.py and hid_devices.py should be added back into this list, but currently they error when compiled with mypyc.